packages/flutter_tools/lib/src/android/android_studio.dart

```diff
@@ -50,11 +67,7 @@ class AndroidStudio implements Comparable<AndroidStudio> {
     if (versionString != null)
       version = Version.parse(versionString);
 
-    final String plistValue = iosWorkflow.getPlistValueFromFile(
-      plistFile,
-      null,
-    );
-    final String pathsSelectorValue = _pathsSelectorMatcher.stringMatch(plistValue).split('=').last.trim().replaceAll('"', '');
+    final String pathsSelectorValue = _pathsSelectorMatcher.stringMatch(plistValue)?.split('=')?.last?.trim()?.replaceAll('"', '');
     return AndroidStudio(studioPath, version: version, pathsSelectorPath: pathsSelectorValue);
   }
 
```

Comment:
How does this toolbox work on Linux/Windows? Do we need to do anything and/or leave comments here?

Reply:
I'll check the behavior on Linux/Windows.

Reply:
In Linux/Windows, the original logic works fine.
We don't need to do more job to deal with JetbrainsToolsbox related logic.